backend/api/rooms.js

```diff
@@ -0,0 +1,28 @@
+const handler = require('../handler/handler');
+
+module.exports = (req, res) => {
+    // eslint-disable-next-line no-console
+    console.log('INFO: /rooms has been called.');
+    res.setHeader('Access-Control-Allow-Origin', '*');
+
+    let data = handler.getData();
+
+    let returnRooms = [];
+    for (var i = 0; i < data.rooms.length; i++) {
```

Comment:
I think we can wrap this in a map function

```suggestion
    const returnRooms = data.rooms.map( ...
```